lib/stm32cubemx_init/library.json

```diff
@@ -1,12 +1,9 @@
 {
   "name": "stm32cubemx_init",
   "description": "STM32CubeMX generated files",
+  "frameworks": ["stm32cube"],
   "build": {
-    "srcFilter": [
-      "+<Src>",
-      "+<Inc>",
-      "-<Src/system_stm32*.c>"
-    ],
+    "srcFilter": "+<*.c>, -<system_stm32*.c>",
     "flags": [
```

Comment:
I've checked twice, this change breaks build under linux. Error is the same as under win without this change (not found files to compile => linker can't find `main` in the end)